CHANGELOG_PENDING.md

```diff
@@ -1,7 +1,5 @@
 ### Improvements
 
-- Codegen: pluginDownloadURL is now supported through the automatically generated build.gradle
```

Comment:
I think we want to keep the changes in this file, as it'll be used by goreleaser when creating the GH release. But it shouldn't be a problem. We can edit the GH release's description after the fact.

Reply:
Sorry I thought it was a manual step.